Add `BN Mentorship: Cycle 1 Sign-ups Open` news post (#10058)
ppy/osu-wiki: news/2023/2023-09-19-bn-mentorship-cycle-1.md

After extensive feedback gathered from the test phase, we're proud to open sign-ups for the first official cycle of the BN Mentorship Program!

But wait, you ask… What the *Haitch E. Double Hockey Sticks* is this "BN Mentorship Program"?

Why, it is none other than a special modding course run by the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (us, the authors of this post) designed to get a modder's knowledge and skill up to what we consider to be the standard skill level of a Beatmap Nominator. The course will teach you how use a bunch of super helpful tools, plus the most important ranking rules you need to watch out for, as well as lots, and we mean *lots* of modding sessions to forge your skills with hands-on practice.

> Unlike the prototype cycle, the mentors will be picking the mentees this time around. We'll take into consideration your previous BN applications (if you've had any), your recent mods and activity, if you have the required 200 *[kudosu!](/wiki/Modding/Kudosu)* to be a candidate, as well as your general *vibe*. Don't be discouraged if you don't get picked, as this is built to be a cyclical thing, which means there's always the next cycle to look forward to.

Reviewer: re: eph's review, it may be sufficient to just omit this comparison

```suggestion
At the start of the cycle, mentors will be picking the mentees. We'll take into consideration your previous BN applications (if you've had any), your recent mods and activity, if you have the required 200 *[kudosu!](/wiki/Modding/Kudosu)* to be a candidate, as well as your general *vibe*. Don't be discouraged if you don't get picked, as this is built to be a cyclical thing, which means there's always the next cycle to look forward to.
```
